app/src/main/java/com/app/foodforyou/model/DietListResponse.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Date;
 
 // Response API
-public class RecommendDietListResponse {
+public class DietListResponse {
     private int cntntsNo; // 컨텐츠 번호
```

Comment:
Please write descriptive variable names to provide more clarity :)

Reply:
Thank you for your feedback! I'm trying to change code using Retrofit. then object's variable name should be same with API response's variable name. so I should keep this now.. Instead, I'll add English comment. :)